pkg/cmd/attestation/download/download.go

```diff
@@ -23,7 +23,7 @@ func NewDownloadCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Comman
 		Args:  cmdutil.ExactArgs(1, "must specify file path or container image URI, as well as one of --owner or --repo"),
 		Short: "Download an artifact's attestations for offline use",
 		Long: heredoc.Docf(`
-			### NOTE: This feature is currently in beta, and subject to change.
+   	  		### NOTE: This feature is currently in beta, and subject to change.
```

Comment:
this is what i get for trying to revert this change using the web editor 🤦‍♀️ 🤦‍♀️ 🤦‍♀️